### DISCUSSION TITLE
Proposed Outline Revision for Unit 7. Video & Video Processing

### DISCUSSION THREAD

**Discussion by wo0jun** [huggingface/computer-vision-course#348] [07. Video & Video Processing]
Hello! @cjfghk5697 , @mreraser, @1kmmk1 and I are working on completing the Video & Video Processing unit. Before we begin, we'd like to modify the outline as follows to ensure consistency with other units:

### Intro
* Introduction
* Video processing basics
* Video Tasks and Models
### Deep learning architecture
   * CNN based Video models
   * RNN(with Attention) based Video models
   * Transformer based Video models
   * Multimodal based Video models
### Applications
   * Load and explore a video dataset
   * Pre-trained models for video classification
   * Fine-tuning a model for video classification

What do you think about this outline? @johko

**Comment by johko** [comment]
Hey @jungnerd, thanks for the proposal. Looks very good as far as I can see. I like the approach of building up from CNN to Multimodal and also the Applications part :+1:
